test/martha/martha_v3.test.js

```diff
@@ -63,7 +65,16 @@ const mockResponse = () => {
             this.statusCode = s;
             return this;
         },
-        send: sinon.stub(),
+        send: sinon.mock('send').once().callsFake(function setBody(body) {
+            // Express will effectively JSON.stringify objects passed to send, which is perfect for
+            // sending over the wire and deserializing into a simple object on the other end.
+            // We want a similar effect here, where all we get are the object properties (and
+            // specifically no class instance details) so that we can make comparisons against
+            // simple objects. Specifically, this takes care of cases where we "send" a
+            // FailureResponse.
+            this.body = { ...body };
+            return this;
+        }),
```

Comment:
This lets us check `response.body` instead of `{ ...response.send.lastCall.args[0] }` in our tests.